purchase_manual_delivery/views/purchase_order_views.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_view_form_inherit" model="ir.ui.view">
+        <field name="name">purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//header/button[@name='action_rfq_send']" position="after">
+                <field name="pending_to_receive" invisible="1" />
+                <button
+                    name="%(action_create_manual_stock_picking)d"
+                    string="Create Incoming Shipment"
+                    type="action"
+                    class="btn-primary"
+                    attrs="{'invisible': ['|', ('state', '!=', 'purchase'), ('pending_to_receive', '=', False)]}"
+                />
+            </xpath>
+            <button id="draft_confirm" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </button>
+            <button id="draft_confirm" position="after">
+                <button
+                    name="button_confirm_manual"
+                    type="object"
+                    states="draft"
+                    string="Confirm Order"
+                    id="draft_confirm_manual"
+                />
```

Comment:
Don't understand the need to create a separate method and button for this. Why not just update the context in button_confirm directly.

Reply:
Non-important feedback. Still valid maybe though.